proteus/Transport.py

```diff
@@ -2714,6 +2712,8 @@ def calculateElementJacobian(self,skipMassTerms=False):
                     if self.timeIntegration.diffusionIsImplicit[ci]:
                         if self.numericalFlux == None or self.numericalFlux.mixedDiffusion[ci] == False:
                             if self.sd:
+                                import pdb
```

Comment:
probably want to delete these from committed code. they will cause travis to fail.